traitsui/qt4/check_list_editor.py

```diff
@@ -137,6 +145,13 @@ class CustomEditor(SimpleEditor):
         boxes.
     """
 
+    #: List of tuple(signal, slot) to be disconnected in dispose
+    _connections_to_dispose = List(Tuple(Any, Callable))
+
+    #: List of tuple(signal, slot) to be disconnected while rebuilding the
+    #: editor.
+    _connections_to_rebuild = List(Tuple(Any, Callable))
+
```

Comment:
I will add a `init` here to do exactly what `SimpleEditor` does, but call `EditorWithList.init(self)` directly. This will read correctly with `dispose` and shield `CustomEditor.init` from changes in `SimpleEditor.init`.  The `init` and `dispose` of these two classes should be decoupled.